policy_definitions/network_security/policy_definition_qby_VnetNamingConform.json

```diff
@@ -0,0 +1,161 @@
+{
+  "type": "Microsoft.Authorization/policyDefinitions",
+  "name": "QBY-Deny-Vnet-Subnet-NSG",
+  "properties": {
+    "displayName": "Vnet Subnet NSG Convention",
```

Comment:
The displayname doesn't help to understand what this policy does. The main focus is the associated NSG.

Reply:
got adjusted